website/content/en/docs/deployment/pipeline-profiles.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "Create Profiles with IAM role"
+description = "Use AWS IAM roles for service accounts with Kubeflow Profiles"
+weight = 70
++++
+
+In a multi tenant Kubeflow installation, the pods created by pipelines workflow and the pipelines frontend services run in an user profile namespace. The service account (`default-editor`) used for these pods needs permissions for the S3 bucket used by pipelines to read and write artifacts from S3. When using IRSA (IAM roles for service accounts) as your pipeline S3 credential option, any additional profiles created as part of a multi-user deployment besides the preconfigured `kubeflow-user-example-com` will need to be configured with permissions to S3 bucket using IRSA.
+
+The `default-editor` SA needs to be annotated with an IAM role with sufficient permissions to access your S3 Bucket to run your pipelines. In the below steps we will be configuring a profile an IAM role with restricted access to a specific S3 Bucket using the `AwsIamForServiceAccount` plugin for Profiles. To learn more about Profiles read the [Profiles component guide]({{< ref "/docs/component-guides/profiles.md" >}}).
+
+>Note: If you choose to run your pipeline with a service account other than the default which is `default-editor`, you must make sure to annotate that service account with an IAM role with sufficient S3 permissions.
+
+#### Admin considerations
+
+- Kubeflow admins will need to create an IAM role for each Profile with the desired scoped permissions.
+- A `default-editor` SA exists in every Profile's namespace and will be annotated with the role ARN created for the profile. Pods annotated with the SA name will be granted the Profile role permissions.
+- The `default-editor` SA is used by various services in Kubeflow to launch resources in Profile namespaces. However, not all services do this by default.
+
+## Create a Profile
+
+After installing Kubeflow on AWS with one of the available [deployment options]({{< ref "/docs/deployment" >}}), you can configure Kubeflow Profiles with the following steps:
+
+1. Define the following environment variables:
+
+   ```bash
+   export CLUSTER_NAME=<your cluster name>
+   export CLUSTER_REGION=<your region>
+   export S3_BUCKET=<your s3 bucket>
```

Comment:
- Need to highlight S3 bucket name should match the one used by pipelines. Edit this text and also put it in text of the step
- Instead of `export CLUSTER_NAME=<your cluster name>`, following might be better UX so user can copy+paste and start input their values directly
```
# Your cluster name
export CLUSTER_NAME=
```